test/reducers/user.spec.js

```diff
@@ -0,0 +1,22 @@
+import { expect } from 'chai';
+import reducer from '../../app/reducers/user';
+
+describe('reducers', () => {
+
+  describe('user', () => {    
+    
+    it('should handle USER_LOGIN', () => {
+      const action = { 
+        type: 'USER_LOGIN',
+        payload: {
+          username: 'John Doe', 
+          loggedIn: true 
+        }
+      };
+      const test = Object.assign({}, action.payload);
+      expect(reducer({}, action)).to.deep.equal(test);
```

Comment:
Nevermind I dont think that's a problem. State is always being set to `{}` so theres no chance of other actions polluting that state object which would fail the test.